ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDb.java

```diff
@@ -417,6 +395,17 @@ Map<String, DataSource> getDataSources() {
         .collect(Collectors.toMap(Map.Entry::getKey, e -> e.getValue().getDataSource()));
   }
 
+  /**
+   * Build the version for Named Blob row based on timestamp and uuid postfix.
+   * @return a long number whose rightmost 5 digits are uuid postfix, and the remaining digits are current timestamp
+   */
+  private static long buildVersion() {
+    long currentTime = System.currentTimeMillis();
+    UUID uuid = UUID.randomUUID();
```

Comment:
how about using datacenter id - lva, ltx, lor instead of randomUUID so that we are more deterministic in execution ?

Reply:
I didn't use datacenter id to avoid concurrency within a datacenter (eg multiple concurrent requests for the same named blob in datacenter lva).